sources/web/datalab/polymer/components/auth-panel/auth-panel.ts

```diff
@@ -47,7 +47,10 @@ class AuthPanel extends Polymer.Element {
 
   ready() {
     super.ready();
-    GapiManager.loadGapi(this._signInChanged.bind(this));
+    GapiManager.listenForSignInChanges(this._signInChanged.bind(this))
+      .catch(() => {
+        // TODO: handle errors authenticating
```

Comment:
I'm not sure we want to do this. So far, having a client ID to make this auth possible is only an option, so it shouldn't be an error.